package.json

```diff
@@ -35,21 +20,107 @@
     "examples/shortcodes",
     "examples/terminal"
   ],
+  "scripts": {
+    "bootstrap": "yarn && lerna bootstrap && lerna link --force-local",
+    "build": "r -f cjs,esm -o dist --exclude 'babel-plugin-*,create-mdx,*loader,mdx,parcel-plugin-mdx,remark-*,runtime,*util'",
+    "clean": "lerna exec \"rm -rf node_modules\"",
```

Comment:
should clean also remove `dist` folders?

Reply:
I didn't change it, it's just sort by `prettier-plugin-pkg`.